<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CClientActionsWidget</class>
 <widget class="QWidget" name="CClientActionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbxActions">
     <property name="title">
      <string>Мероприятия</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Фильтр</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Категория</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QComboBox" name="cbxCategory"/>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="tbnCategoryClear">
              <property name="icon">
               <iconset resource="../clinet_res.qrc">
                <normaloff>:/clientImages/clear.png</normaloff>:/clientImages/clear.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Исполнитель</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QComboBox" name="cbxPerformer"/>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="tbnPerformerCLear">
              <property name="icon">
               <iconset resource="../clinet_res.qrc">
                <normaloff>:/clientImages/clear.png</normaloff>:/clientImages/clear.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Площадка</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QComboBox" name="cbxPlace"/>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="tbnPlaceClear">
              <property name="icon">
               <iconset resource="../clinet_res.qrc">
                <normaloff>:/clientImages/clear.png</normaloff>:/clientImages/clear.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Дата</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLineEdit" name="leDate">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="tbnDate">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QToolButton" name="tbnDateClear">
              <property name="icon">
               <iconset resource="../clinet_res.qrc">
                <normaloff>:/clientImages/clear.png</normaloff>:/clientImages/clear.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="4">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QToolButton" name="tbnFilterClear">
             <property name="icon">
              <iconset resource="../clinet_res.qrc">
               <normaloff>:/clientImages/clear.png</normaloff>:/clientImages/clear.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTreeWidget" name="twActions">
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>150</number>
        </attribute>
        <column>
         <property name="text">
          <string>id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Название</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Исполнитель</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата проведения</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Площадка</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Категория</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbnUpdate">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../clinet_res.qrc"/>
 </resources>
 <connections/>
</ui>
